ext/gd/libgd/gd.c

```diff
@@ -382,51 +382,6 @@ static float HWB_Diff (int r1, int g1, int b1, int r2, int g2, int b2)
 }
 
 
-#if 0
```

Comment:
libgd is running it's own merging process, better ask @pierrejoye and @remicollet 

Thanks.